reactor-netty-examples/src/main/java/reactor/netty/examples/http/snoop/HttpSnoopClient.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2023 VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.examples.http.snoop;
+
+import io.netty.handler.codec.http.HttpHeaderNames;
+import io.netty.handler.codec.http.HttpHeaderValues;
+import reactor.core.publisher.Mono;
+import reactor.netty.ByteBufFlux;
+import reactor.netty.http.client.HttpClient;
+
+/**
+ * @author Kun.Long
+ * a http client demo that sends two requests to the http snoop server and then wait for the completion of the two requests
+ * @see HttpSnoopServer
+ **/
+public class HttpSnoopClient {
+
+	private final HttpClient client = HttpClient.create().wiretap(false);
+
+	public static void main(String[] args) {
+		HttpSnoopClient httpSnoopClient = new HttpSnoopClient();
+		httpSnoopClient.client.warmup().block();
```

Comment:
Do we want to use `warmup` functionality? The example should be working even without warmup.